build/buildBuildImages.sh

```diff
@@ -300,11 +300,11 @@ function buildLtsVersionsImage() {
 		.
 }
 
-function buildFullImage() {
+function buildAllImages() {
 	buildLtsVersionsImage
 
 	echo
-	echo "-------------Creating full build image-------------------"
```

Comment:
So this method doesn't appear to be building all of the build images -- it looks like it's just building `$BUILD_IMAGES_DOCKERFILE`, which is the `latest` build image. It would probably make sense to change the method to `buildLatestImage` and update the comment on this line

Reply:
Updated. Thanks!